maud/src/lib.rs

```diff
@@ -7,7 +9,15 @@
 
 #![doc(html_root_url = "https://docs.rs/maud/0.22.2")]
 
-use std::fmt::{self, Write};
+// So that unit tests can use `std`
+#[cfg(test)]
+#[macro_use]
+extern crate std;
```

Comment:
Can we remove this? This module doesn't define any unit tests afaik.

Reply:
This was updated to re-enable std for rocket and iron support.